packages/core/contracts/oracle/implementation/EmergencyProposer.sol

```diff
@@ -81,18 +79,15 @@ contract EmergencyProposer is Ownable, Lockable {
      * @param _token the ERC20 token that the quorum is in.
      * @param _quorum the tokens needed to propose an emergency action..
      * @param _governor the governor contract that this contract makes proposals to.
-     * @param _finder the finder contract used to look up addresses.
```

Comment:
We also need to remove `currentId` above. (I actually had this as a staged change locally, but noticed it never got committed 🤦 ).